llvm/test/Instrumentation/HWAddressSanitizer/kernel.ll

```diff
@@ -1,3 +1,4 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
```

Comment:
update_test_checks does not work realy well with multiple refixes CHECK,NOOFFSET,MATCH-ALL

In a separate PR can you regenerate and assign single prefix per test case?